bin/local-env/install-wordpress.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+# Exit if any command fails.
+set -e
+
+# Common variables.
+WP_DEBUG=${WP_DEBUG-true}
+SCRIPT_DEBUG=${SCRIPT_DEBUG-true}
+WP_VERSION=${WP_VERSION-"latest"}
+
+# Include useful functions
+. "$(dirname "$0")/includes.sh"
+
+# Make sure Docker containers are running
+dc up -d >/dev/null 2>&1
+
+# Get the host port for the WordPress container.
+HOST_PORT=$(dc port $CONTAINER 80 | awk -F : '{printf $2}')
+
+# Wait until the Docker containers are running and the WordPress site is
+# responding to requests.
+echo -en $(status_message "Attempting to connect to WordPress...")
+until $(curl -L http://localhost:$HOST_PORT -so - 2>&1 | grep -q "WordPress"); do
+    echo -n '.'
+    sleep 5
+done
+echo ''
+
+# If this is the test site, we reset the database so no posts/comments/etc.
+# dirty up the tests.
+if [ "$1" == '--reset-site' ]; then
+	echo -e $(status_message "Resetting test database...")
+	wp db reset --yes --quiet
+fi
+
+if [ ! -z "$WP_VERSION" ] && [ "$WP_VERSION" != "latest" ]; then
+	# Potentially downgrade WordPress
+	echo -e $(status_message "Downloading WordPress version $WP_VERSION...")
+	wp core download --version=${WP_VERSION} --force --quiet
+fi
+
+# Install WordPress.
+echo -e $(status_message "Installing WordPress...")
+wp core install --title="$SITE_TITLE" --admin_user=admin --admin_password=password --admin_email=test@test.com --skip-email --url=http://localhost:$HOST_PORT  --quiet
+
+# Create additional users.
+echo -e $(status_message "Creating additional users...")
+wp user create editor editor@example.com --role=editor --user_pass=password --quiet
```

Comment:
If you keep databse, these fail. So this change needs to me made. 
```suggestion
wp user create editor editor@example.com --role=editor --user_pass=password --quiet || true
```

Reply:
Did something similar in 8608ac4bd50767f904632e113a368f177ae3971a